realm/realm-library/src/main/cpp/CMakeLists.txt

```diff
@@ -138,7 +138,7 @@ set(CMAKE_CXX_FLAGS_DEBUG "-ggdb -Og -DNDEBUG")
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${REALM_COMMON_CXX_FLAGS} ${WARNING_CXX_FLAGS} ${ABI_CXX_FLAGS}")
 
 # Set link flags
-set(REALM_LINKER_FLAGS "")
+set(REALM_LINKER_FLAGS "-Wl,--wrap,memmove -Wl,--wrap,memcpy")
```

Comment:
I have seen a few stack traces with `memcpy`. But it can be that `memcpy` calls `memmove` in certain cases.